NetcoreBootstrap/Startup.cs

```diff
@@ -50,10 +50,10 @@ public void ConfigureServices(IServiceCollection services)
                                             options.LoginPath = "/Account/Login";
                                             options.AccessDeniedPath = "/Account/AccessDenied";
                                         });
-            services.AddAuthentication().AddFacebook(facebookOptions => {
-                                            facebookOptions.AppId = Configuration["Authentication:Facebook:AppId"];
-                                            facebookOptions.AppSecret = Configuration["Authentication:Facebook:AppSecret"];
-                                        });
+            // services.AddAuthentication().AddFacebook(facebookOptions => {

```

Comment:
why do you comment this?

Reply:
I commented this because the project would not run, since i hadn't the facebook credentials available